src/frontend/src/pages/TeamsPage/TeamMembersPageBlock.tsx

```diff
@@ -7,118 +7,157 @@ import { Autocomplete, Grid, IconButton, TextField } from '@mui/material';
+    }
+  };
 
-  const editingView = (
-    <PageBlock title={'People'} headerRight={editButtons}>
-      <Grid container spacing={1}>
-        <Grid item xs={12}>
-          <DetailDisplay label="Head" content={fullNamePipe(team.head)} />
+  const submitMembers = async () => {
+    try {
+      await setTeamMembersMutateAsync(members.map((member) => parseInt(member.id)));
+      setIsEditingMembers(false);
+    } catch (error) {
+      error instanceof Error && toast.error(error.message);
+    }
+  };
+
+  const editingHeadView = (
+    <Grid container direction={'row'} pt={2} pl={1}>
+      <Grid item xs={8} md={10} lg={11}>
+        <NERAutocomplete
+          id="head-autocomplete"
+          options={headOptions}
+          onChange={(_event, newValue) => newValue && setHead(newValue)}
+          filterSelectedOptions
+          size="small"
+          placeholder="Select a User"
+          value={head}
+          label={'Head'}
+        />
+      </Grid>
+      <Grid container direction={'row'} xs={4} md={2} lg={1}>
+        <Grid item>
+          <IconButton children={<Save />} onClick={submitHead} />
         </Grid>
-        <Grid item xs={12}>
-          <Autocomplete
-            isOptionEqualToValue={(option, value) => option.id === value.id}
-            filterSelectedOptions
-            multiple
-            id="tags-standard"
-            options={options}
-            value={members}
-            onChange={(_event, newValue) => setMembers(newValue)}
-            getOptionLabel={(option) => option.label}
-            renderInput={(params) => (
-              <TextField {...params} variant="standard" label="Members" placeholder="Select A User" />
-            )}
-          />
+        <Grid item>
+          <IconButton children={<Cancel />} onClick={() => setIsEditingHead(false)} />
         </Grid>
       </Grid>
-    </PageBlock>
+    </Grid>
   );
 
-  const nonEditingView = (
-    <PageBlock
-      title={'People'}
-      headerRight={hasPerms ? <IconButton onClick={() => setIsEditingMembers(true)} children={<Edit />} /> : null}
-    >
-      <Grid container spacing={1}>
-        <Grid item xs={12}>
-          <DetailDisplay label="Head" content={fullNamePipe(team.head)} />
-        </Grid>
-        <Grid item xs={12}>
-          <DetailDisplay label="Leads" content={team.leads.map((lead) => fullNamePipe(lead)).join(', ')} />
+  const editingMembersView = (
+    <Grid container direction={'row'} pl={1}>
+      <Grid item xs={8} md={10} lg={11}>
+        <Autocomplete
+          isOptionEqualToValue={(option, value) => option.id === value.id}
+          filterSelectedOptions
+          multiple
+          id="tags-standard"
+          options={memberOptions}
+          value={members}
+          onChange={(_event, newValue) => setMembers(newValue)}
+          getOptionLabel={(option) => option.label}
+          renderInput={(params) => <TextField {...params} variant="standard" label="Members" placeholder="Select A User" />}
```

Comment:
For members, keep the "Members:" typography label and put the autocomplete and buttons on the line below instead of this label prop